### DISCUSSION TITLE
What is JellyChain1?

### DISCUSSION THREAD

**Discussion by Cotopia-Inc** [JellyChain1/JellyChain1#2] [General]
What is JellyChain1?
JellyChain1 is a blockchain network platform that supports smart contracts and can act as a payment system/cryptocurrency. It is originally and still is made for experimental and educational purposes, you can have a brief look at its core ideas in this repository.

**Comment by LifeCoachRay** [comment]
I will contribute in any way I could.

**Reply by Cotopia-Inc** [reply]
Thank You for your support. Your GitHub is very good.

**Comment by HeartBreakKid8** [comment]
I got the blockchain connected and running. I really enjoy being a miner on this chain. The process is smoother and much easier to mine coins. The average person can now mine transactions on the chain.

**Reply by Cotopia-Inc** [reply]
We believe that miners should not have to go bankrupt or have to deal with complicated equipment and programs to run JellyChain1. So We are glad you are enjoying your experience. Please feel free to contribute in any way you can.

**Reply by BotGame22** [reply]
so far it have not cost me anything out of pocket to run this blockchain and get rewarded.

**Reply by billyjo111** [reply]
I love it

**Comment by BotGame22** [comment]
I downloaded my copy of the chain and it runs well so far. tonight I will try to connect a simple app to it. hope everything goes well with that.

**Comment by HeartBreakKid8** [comment]
JellyChain1 is ground breaking. All you do is fire it up and it will do the rest. I run 3 node side by side and I can still run programs in the background. Thank you JellyChain1 for this awesome tech. I will do my best to help where needed
